src/components/Person/Person.jsx

```diff
@@ -1 +1,55 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const getmaried = [];
```

Comment:
1. use object destructing for `person`
```suggestion
  const { name, age, sex, isMarried, partnerName } = person;
```